kubernetes-loading-container.el

```diff
@@ -15,7 +15,7 @@
   (cond
    ((null vector)
     on-loading)
-   ((and vector (seq-contains-p vector elem))
+   ((and vector (if (fboundp 'seq-contains-p) (seq-contains-p vector elem) (seq-contains vector elem)))
```

Comment:
Trying this: https://github.com/watofundefined/doom-emacs-private/blob/6a66041a711e2b22f0b3034bcaa1b4fc419243e8/modules/wat-utils.el#L21-L23